packages/dialect-react-ui/components/Avatar/index.tsx

```diff
@@ -33,18 +35,25 @@ const CardinalAvatar = ({
   placeholder?: React.ReactNode;
   className?: string;
 }) => {
-  const { addressImage, loadingImage } = useAddressImage(connection, address);
+  const reverseEntry = useSWR([connection, address], getReverseEntry);
```

Comment:
I think we should abstract this into a custom hook inside react-ui package, wdyt?
Similar to `useAddressImage`, but our own.

Reply:
@fsher what do you think about this being a part of the member object in the `react` package? feels agnostic to react-ui specifically, could be reused e.g. in react native, etc.

Reply:
should clarify: I'm on board with a quick implementation for now, fwiw

Reply:
That's a good question and I think you're right. It makes sense, since react provides the data and react-ui renders it. So yeah, probably better to move to `react`. Agree.